pom.xml

```diff
@@ -35,6 +35,11 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.modules</groupId>
+      <artifactId>instance-identity</artifactId>
+      <version>1.4</version>
+    </dependency>
```

Comment:
I think you need to use either `<optional>true</optional>` or `<scope>provided</scope>` to prevent this from getting bundled in `WEB-INF/lib/*.jar`.